cdap-ui/templates/ETLBatch/Elasticsearch.json

```diff
@@ -38,6 +38,21 @@
           "properties": {
             "width": "large"
           }
+        },
+        "schema" : {
+          "widget": "schema",
+          "label": "Schema",
+          "description" : "Specify the schema for the table",
+          "schema-types" : [ "boolean", "int", "long", "float", "double", "bytes", "string" ],
+          "schema-default-type" : "string"
+        },
+        "query" : {
+          "widget": "textbox",
+          "label": "Query",
+          "description": "The query used to select record from the specified index/type. e.g. ?q=* will select all records",
```

Comment:
```
"The query used to select records from the specified index/type; for example, ?q=* will select all records",
```
Best to avoid abbreviations and Latin in comments and descriptions.